README.md

```diff
@@ -51,3 +69,60 @@ using the following command:
 ```shell
 $ ENABLE_WEBHOOKS=false make run install
 ```
+
+## Testing the operator using Kind on Fedora 38
```

Comment:
Could we make this more generic? The only thing specific to Fedora 38 is `kind` and `kubectl` install. The rest is universal (I have macOS).

So perhaps the main title could be: "Testing the operator using Kind"

And then under it would be a subtitle "Installing kind and kubectl on Fedora 38".

I could also provide a similar section for macOS:

Typically you will use `podman machine`.

So something like:
```
$ brew install podman
$ podman machine init
$ podman machine set --cpus 2
$ podman machine start
$ brew install kind
$ export KIND_EXPERIMENTAL_PROVIDER=podman
```
For kubectl I would probably just link the docs: https://kubernetes.io/docs/tasks/tools/